CMSgov/cms-carts-seds
# MDCT-192: Uncertify CARTS Report (Modal Dialog)

# Description

Closes [MDCT-192](https://qmacbis.atlassian.net/browse/MDCT-192). An approver can now click on `Uncertify` to uncertify a report using a dialog that will pop up and ask that user to confirm.

## How to test

`./dev local`

Log in as Admin (functionally, an Approver) and click `Uncertify` for one of the reports. The following should happen:

- A modal dialog magically appears
- You are able to close the dialog using the magical **Close** button on the top-right
- If you click on **Yes, Uncertify Report**, the page magically reloads
- Upon reload, the report status has changed to "In Progress" ... magically

One more _magically_ for good measure 🪄 

## Dependencies

N/A

# Get it done

Now that the PR is open this is what happens next

## Code authors checklist


## Reviewers Checklist (Two different people)


## Assignee
